### PR TITLE
split rpm tests out so they are easier to exclude for mac buildkite runs

### PR DESCRIPTION
**Pull request by aiuto**
Doing this now will make it easier to get write the test specification for buildkite.
We can't do any rpm tests on macos or windows until #29 is fixed.

### CODE REVIEW

**Comment by aiuto** [comment]
This is really to get in shape to add more windows tests.

File: pkg/tests/BUILD [hunk 1]
```diff
@@ -3,12 +3,68 @@
 licenses(["notice"])
```
Comment (nacl) [inline pkg/tests/BUILD:3]: There's no copyright statement in this file, but since this is preexisting, not a problem for this review.

**Review by nacl — APPROVED** [review]
Nice.  Comments are super minor.

File: pkg/tests/rpm/BUILD [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (aiuto) [reply pkg/tests/rpm/BUILD:1]: whoops. I will fix that.

**Pull request merged**